metaflow/cli.py

```diff
@@ -606,11 +694,124 @@ def run(obj,
     write_latest_run_id(obj, runtime.run_id)
     write_run_id(run_id_file, runtime.run_id)
 
+    # import pickle
+    #
+    # with open('graph.pkl', 'wb') as gh:
+    #     pickle.dump(obj.graph, gh)
+
     parameters.set_parameters(obj.flow, kwargs)
     runtime.persist_parameters()
     runtime.execute()
 
 
+@parameters.add_custom_parameters
+@cli.command(help='Run the initial part of the workflow (i.e., save parameters locally).')
+@common_run_options
+@click.option('--namespace',
+              'user_namespace',
+              default=None,
+              help="Change namespace from the default (your username) to "
+                   "the specified tag. Note that this option does not alter "
+                   "tags assigned to the objects produced by this run, just "
+                   "what existing objects are visible in the client API. You "
+                   "can enable the global namespace with an empty string."
+                   "--namespace=")
+@click.pass_obj
+def pre_start(obj,
+        tags=None,
+        max_workers=None,
+        max_num_splits=None,
+        max_log_size=None,
+        decospecs=None,
+        run_id_file=None,
+        user_namespace=None,
+        **kwargs):
+
+    if namespace is not None:
+        namespace(user_namespace or None)
+    before_run(obj, tags, decospecs + obj.environment.decospecs())
```

Comment:
style: leave an empty line after if or else block (or any block).  Helps delineate code blocks and reduce chance of bugs (where a developer would think that the following line is in the same block by accident, this has happened).

Reply:
Done